tools/cli/internal/cli/merge/merge_test.go

```diff
@@ -56,6 +58,31 @@ func TestSuccessfulMerge_Run(t *testing.T) {
 	}
 }
 
+func TestNoBaseSpecMerge_PreRun(t *testing.T) {
+	externalPaths := []string{"external.json"}
+	opts := &Opts{
+		outputPath:    "foas.json",
+		externalPaths: externalPaths,
+	}
+
+	err := opts.PreRunE(nil)
+	require.Error(t, err)
+	require.EqualError(t, err, fmt.Sprintf("no base OAS detected. "+
+		"Please, use the flag %s to include the base OAS", flag.Base))
+}
+
+func TestNoExternalSpecMerge_PreRun(t *testing.T) {
```

Comment:
[style] test should be `Test[Struct]_[MethodName]` if you have "scenarios" for your test use a matrix testing with names in the matrix or a more imperative `t.Run("my case"...`

In IDEA editors you can cmd+shift+t and ask it to generate the above
![Screenshot 2024-03-15 at 13 48 47](https://github.com/mongodb/openapi/assets/461027/73caf0ac-9927-47bf-b196-bad013f800bf)

Reply:
oh this is helpful, thanks!